packages/flutter_tools/test/commands/packages_test.dart

```diff
@@ -129,11 +146,16 @@ class MockProcessManager implements ProcessManager {
 }
 
 class MockProcess implements Process {
+  MockProcess();
```

Comment:
this constructor shouldn't be necessary?

Reply:
Was it meant to be `const`?